src/components/Stats/index.jsx

```diff
@@ -21,11 +30,11 @@ export default function Stats() {
   }, []);
 
   return (
-    <section>
+    <div className="stats-container" style={{ width: '100vw' }}>
+      <button onClick={toggleHelper} type="button">{toggle ? 'Visit Globe' : 'Visit Graph'}</button>
```

Comment:
just for next time, another way to do this is `onClick={setToggle((prev) => !prev)}`